### DISCUSSION TITLE
Excel doesn't recognize date format of exported data

### DISCUSSION THREAD

**Discussion by SirParser** [mini-software/MiniExcel#416] [Q&A]
Excel doesn't recognize dates imported from datatable as proper dates. They are treated as text and left alligned. That also causes the date filter not to group the dates into years, months, days etc.
Is there a way to fix this?

![image](https://user-images.githubusercontent.com/102298858/181261486-d82c3948-82a6-4b1c-8aac-dcb5a609899c.png)
![image](https://user-images.githubusercontent.com/102298858/181262108-4106aebe-d231-409e-9e6c-729b63d85b40.png)
![image](https://user-images.githubusercontent.com/102298858/181262335-fe3ad8c8-48b5-4f28-acf0-d044b1c3ac22.png)

**Comment by mgrivera** [comment]
Hi. 
Although this is an old post, I can see it happening right now. 
I you have your dates as DateOnly in your Net application, the format in Excel is acceptable, even if they don't recognize the cell format. 
Numbers formatting works pretty well but not for dates, at least what I have been seeing recently.

**Reply by michelebastione** [reply]
We apologize for the inconvenience and for the late reply. The problem should be solved now, but please let us know if you're still experiencing the issue.
